### PR TITLE
Set file timestamps in image to epoch + 1 second

### PR DESCRIPTION
**Pull request by chanseokoh**
Short-term solution for #1079.

### CODE REVIEW

File: jib-gradle-plugin/CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ All notable changes to this project will be documented in this file.
 
 - Removed deprecated `jib.jvmFlags`, `jib.mainClass`, `jib.args`, and `jib.format` in favor of the equivalents under `jib.container` ([#461](https://github.com/GoogleContainerTools/jib/issues/461))
 - `jibExportDockerContext` generates different directory layout and `Dockerfile`.
+- File timestamps in the build image are set to 1 second since the epoch (hence 1970-01-01T00:00:01Z); previously they were set to the epoch. ([#1079](https://github.com/GoogleContainerTools/jib/issues/1079))
```
Comment (coollog) [inline jib-gradle-plugin/CHANGELOG.md]: maybe add "to resolve compatibility with <Java 6 applications where file time of 0 means nonexistent"?

File: jib-gradle-plugin/CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ All notable changes to this project will be documented in this file.
 
 - Removed deprecated `jib.jvmFlags`, `jib.mainClass`, `jib.args`, and `jib.format` in favor of the equivalents under `jib.container` ([#461](https://github.com/GoogleContainerTools/jib/issues/461))
 - `jibExportDockerContext` generates different directory layout and `Dockerfile`.
+- File timestamps in the build image are set to 1 second since the epoch (hence 1970-01-01T00:00:01Z) to resolve compatibility with applications on Java 6 or below where the epoch means nonexistent or I/O errors; previously they were set to the epoch. ([#1079](https://github.com/GoogleContainerTools/jib/issues/1079))
```
Comment (loosebazooka) [inline jib-gradle-plugin/CHANGELOG.md]: I think this I/O exception on 0 happens on all versions of java, not just 6- ?

Comment (chanseokoh) [reply jib-gradle-plugin/CHANGELOG.md]: Yeah, my interpretation is that 0 can be a valid timestamp but may also mean an error, so to be sure, you have to use some other means to double-check, if you have to.

**Comment by loosebazooka** [comment]
lgtm

**Pull request merged**